### PR TITLE
refactor(container-kill): Adding the task banner in the container-kill experiment

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

**What this PR does / why we need it**:

- Adding the task banner in the container-kill experiment.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1423 

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [x] Have you added debug messages where necessary? 
* [x] Have you added task comments where necessary? 
* [x] Have you tested the changes for possible failure conditions?
* [x] Have you provided the positive & negative test logs for the litmusbook execution?
* [x] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [x] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [x] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [x] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [x] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [x] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [x] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [x] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ispeakc0de** [comment]
```

2020-04-13 08:57:25.447109 ************ BRACE YOURSELF, EXPERIMENT BEGINS! ************ 

2020-04-13 08:57:28.662101 Step: Gathering Facts 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:2
META: ran handlers

2020-04-13 08:57:29.035091 Step: [PreReq] Identify the chaos util for container-kill experiment 
task path: /experiments/generic/container_kill/container_kill_ansible_prerequisites.yml:1
details => {"changed": true, "checksum": "03b27bf248187df047c75919cd9f1b6a0d89c829", "dest": "./chaosutil.yml", "gid": 0, "group": "root", "md5sum": "9e9d63cc391e78455c2cc7892a37a36c", "mode": "0644", "owner": "root", "size": 51, "src": "/root/.ansible/tmp/ansible-tmp-1586768248.7-264731346899912/source", "state": "file", "uid": 0}

2020-04-13 08:57:29.080301 Step: [PreReq]: Including the chaos util for the container-kill experiment 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:25
details => {"ansible_facts": {"c_util": "/chaoslib/pumba/pod_failure_by_sigkill.yml"}, "ansible_included_var_files": ["/experiments/generic/container_kill/chaosutil.yml"], "changed": false}

2020-04-13 08:57:29.130115 Step: [PreReq]: Constructing the chaos result name 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:30
details => {"ansible_facts": {"c_result": "engine1-container-kill"}, "changed": false}
included: /utils/runtime/update_chaos_result_resource.yml for 127.0.0.1

2020-04-13 08:57:29.376192 Step: [PreReq]: Generate the chaos result CR to reflect SOT (Start of Test) 
task path: /utils/runtime/update_chaos_result_resource.yml:3
details => {"changed": true, "checksum": "0304eb3a4afc5be2e859fae8678c01b3d81e8cb6", "dest": "/utils/runtime/chaos-result.yml", "gid": 0, "group": "root", "md5sum": "ce9d71adf0fe7b5bcea3537dab6d0a59", "mode": "0644", "owner": "root", "size": 314, "src": "/root/.ansible/tmp/ansible-tmp-1586768249.21-82044868865074/source", "state": "file", "uid": 0}

2020-04-13 08:57:30.316561 Step: [PreReq]: Apply the chaos result CR for container-kill experiment 
task path: /utils/runtime/update_chaos_result_resource.yml:13
details => {"changed": true, "cmd": "kubectl apply -f /utils/runtime/chaos-result.yml -n shubham", "delta": "0:00:00.723244", "end": "2020-04-13 08:57:30.296386", "failed_when_result": false, "rc": 0, "start": "2020-04-13 08:57:29.573142", "stderr": "", "stderr_lines": [], "stdout": "chaosresult.litmuschaos.io/engine1-container-kill configured", "stdout_lines": ["chaosresult.litmuschaos.io/engine1-container-kill configured"]}

2020-04-13 08:57:30.415691 Step: [Info]: Display the application information passed via the test job 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:43
details => {
    "msg": [
        "The application info is as follows:", 
        "Namespace    : shubham", 
        "Label        : run=nginx", 
        "Ramp Time    : "
    ]
}
included: /utils/common/status_app_pod.yml for 127.0.0.1

2020-04-13 08:57:30.968682 Step: [Status]: Checking whether application pods are in running state 
task path: /utils/common/status_app_pod.yml:2
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -n shubham -l run=nginx -o custom-columns=:.status.phase --no-headers", "delta": "0:00:00.366974", "end": "2020-04-13 08:57:30.947665", "rc": 0, "start": "2020-04-13 08:57:30.580691", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

2020-04-13 08:57:31.666717 Step: [Status]: Checking whether application containers are in running state 
task path: /utils/common/status_app_pod.yml:9
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pod -n shubham -l run=nginx --no-headers -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | tr ' ' '\\n' | uniq", "delta": "0:00:00.555482", "end": "2020-04-13 08:57:31.646339", "rc": 0, "start": "2020-04-13 08:57:31.090857", "stderr": "", "stderr_lines": [], "stdout": "true", "stdout_lines": ["true"]}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 08:57:32.161118 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.314508", "end": "2020-04-13 08:57:32.143245", "rc": 0, "start": "2020-04-13 08:57:31.828737", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T08:57:32+00:00", "stdout_lines": ["2020-04-13T08:57:32+00:00"]}

2020-04-13 08:57:32.229022 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T08:57:32Z"}, "changed": false}

2020-04-13 08:57:32.922668 Step: [Event]: Creating a PreChaosCheck event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "patch", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T08:57:32Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "c3ba3c35-7d64-11ea-8f95-42010a8000cc"}, "kind": "Event", "lastTimestamp": "2020-04-13T08:57:32Z", "message": "AUT is Running successfully", "metadata": {"creationTimestamp": "2020-04-13T08:54:03Z", "name": "event-\"PreChaosCheck\"", "namespace": "shubham", "resourceVersion": "1459", "selfLink": "/api/v1/namespaces/shubham/events/event-%22PreChaosCheck%22", "uid": "53569639-7d64-11ea-8f95-42010a8000cc"}, "reason": "PreChaosCheck", "reportingComponent": "", "reportingInstance": "", "source": {"component": "container-kill-72rqv0-fd2qz"}, "type": "Normal"}}
included: /chaoslib/pumba/pod_failure_by_sigkill.yml for 127.0.0.1

2020-04-13 08:57:33.543040 Step: [Prepare]: Select the application pod 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:3
details => {"changed": true, "cmd": "kubectl get pod -l run=nginx -n shubham -o=custom-columns=:metadata.name --no-headers | shuf | head -1", "delta": "0:00:00.385856", "end": "2020-04-13 08:57:33.513029", "rc": 0, "start": "2020-04-13 08:57:33.127173", "stderr": "", "stderr_lines": [], "stdout": "nginx-6546cd7545-qhpkc", "stdout_lines": ["nginx-6546cd7545-qhpkc"]}

2020-04-13 08:57:33.603278 Step: [Prepare]: Recording the application pod name 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:12
details => {"ansible_facts": {"app_pod": "nginx-6546cd7545-qhpkc"}, "changed": false}

2020-04-13 08:57:34.118666 Step: [Prepare]: Identify the application node name 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:17
details => {"changed": true, "cmd": "kubectl get pod nginx-6546cd7545-qhpkc -n shubham --no-headers -o custom-columns=:spec.nodeName", "delta": "0:00:00.380379", "end": "2020-04-13 08:57:34.102232", "rc": 0, "start": "2020-04-13 08:57:33.721853", "stderr": "", "stderr_lines": [], "stdout": "gke-cassandra-test-default-pool-cf52b41f-bx1h", "stdout_lines": ["gke-cassandra-test-default-pool-cf52b41f-bx1h"]}

2020-04-13 08:57:34.165408 Step: set_fact 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:25
details => {"ansible_facts": {"app_node": "gke-cassandra-test-default-pool-cf52b41f-bx1h"}, "changed": false}

2020-04-13 08:57:34.614650 Step: [Prepare:] Generate a run id if not passed from the engine/experiment 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:29
details => {"changed": true, "cmd": "echo $(mktemp) | cut -d '.' -f 2 | cut -c -6", "delta": "0:00:00.306112", "end": "2020-04-13 08:57:34.587723", "rc": 0, "start": "2020-04-13 08:57:34.281611", "stderr": "", "stderr_lines": [], "stdout": "0eyQNl", "stdout_lines": ["0eyQNl"]}

2020-04-13 08:57:34.666179 Step: set_fact 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:33
details => {"ansible_facts": {"run_id": "0eyqnl"}, "changed": false}

2020-04-13 08:57:34.783296 Step: [Prepare]: Record the application container name 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:52
details => {"ansible_facts": {"a_container": "nginx"}, "changed": false}

2020-04-13 08:57:35.055853 Step: [Prepare]: Patch the chaoslib image to pumba pod 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:68
details => {"changed": true, "checksum": "9b3782aa4666aad3c747264ce72828f676cf20df", "dest": "/chaoslib/pumba/pumba_kube.yml", "gid": 0, "group": "root", "md5sum": "0aa221e197bc092d9a41e87e07b2b2df", "mode": "0644", "owner": "root", "size": 674, "src": "/root/.ansible/tmp/ansible-tmp-1586768254.89-114964666819688/source", "state": "file", "uid": 0}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 08:57:35.539123 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.296013", "end": "2020-04-13 08:57:35.522926", "rc": 0, "start": "2020-04-13 08:57:35.226913", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T08:57:35+00:00", "stdout_lines": ["2020-04-13T08:57:35+00:00"]}

2020-04-13 08:57:35.587050 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T08:57:35Z"}, "changed": false}

2020-04-13 08:57:36.085659 Step: [Event]: Creating a ChaosInject event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "create", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T08:57:35Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "c3ba3c35-7d64-11ea-8f95-42010a8000cc"}, "kind": "Event", "lastTimestamp": "2020-04-13T08:57:35Z", "message": "Injecting container-kill chaos on nginx container of nginx-6546cd7545-qhpkc pod", "metadata": {"creationTimestamp": "2020-04-13T08:57:35Z", "name": "event-\"ChaosInject\"", "namespace": "shubham", "resourceVersion": "1460", "selfLink": "/api/v1/namespaces/shubham/events/event-%22ChaosInject%22", "uid": "d1e59925-7d64-11ea-8f95-42010a8000cc"}, "reason": "ChaosInject", "reportingComponent": "", "reportingInstance": "", "source": {"component": "container-kill-72rqv0-fd2qz"}, "type": "Normal"}}

2020-04-13 08:57:36.616759 Step: [Prepare]: Record the restartCount of application container 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:87
details => {"changed": true, "cmd": "kubectl get pod nginx-6546cd7545-qhpkc -n shubham -o=jsonpath='{.status.containerStatuses[?(@.name==\"nginx\")].restartCount}'", "delta": "0:00:00.386621", "end": "2020-04-13 08:57:36.598974", "rc": 0, "start": "2020-04-13 08:57:36.212353", "stderr": "", "stderr_lines": [], "stdout": "0", "stdout_lines": ["0"]}

2020-04-13 08:57:36.660573 Step: set_fact 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:96
details => {"ansible_facts": {"pod_running_status": ""}, "changed": false}
included: /utils/common/create_chaos_pod.yml for 127.0.0.1 => (item=1)
included: /utils/common/create_chaos_pod.yml for 127.0.0.1 => (item=2)
included: /utils/common/create_chaos_pod.yml for 127.0.0.1 => (item=3)

2020-04-13 08:57:37.361702 Step: [Inject]: Creating the chaos pod in shubham namespace 
task path: /utils/common/create_chaos_pod.yml:3
details => {"changed": true, "cmd": "kubectl apply -f /chaoslib/pumba/pumba_kube.yml -n shubham", "delta": "0:00:00.466951", "end": "2020-04-13 08:57:37.342543", "rc": 0, "start": "2020-04-13 08:57:36.875592", "stderr": "", "stderr_lines": [], "stdout": "pod/pumba-sig-kill-0eyqnl created", "stdout_lines": ["pod/pumba-sig-kill-0eyqnl created"]}
FAILED - RETRYING: [Status]: Checking the running status of the chaos pod (90 retries left).

2020-04-13 08:57:40.464343 Step: [Status]: Checking the running status of the chaos pod 
task path: /utils/common/create_chaos_pod.yml:10
details => {"attempts": 2, "changed": true, "cmd": "kubectl get pods -l name=pumba-sig-kill-0eyqnl -n shubham -o custom-columns=:.status.phase --no-headers", "delta": "0:00:00.386431", "end": "2020-04-13 08:57:40.444077", "rc": 0, "start": "2020-04-13 08:57:40.057646", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

2020-04-13 08:57:40.578034 Step: [Record]: Mark the creation of chaos pod successfully 
task path: /utils/common/create_chaos_pod.yml:43
details => {"ansible_facts": {"pod_running_status": "Running"}, "changed": false}

2020-04-13 08:58:01.207117 Step: [Wait]: Wait till the total chaos duration 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:114
details => {"changed": false, "elapsed": 20, "path": null, "port": null, "search_regex": null, "state": "started"}

2020-04-13 08:58:03.728830 Step: [CleanUP]: Tear down the pumba infrastructure 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:120
details => {"changed": true, "cmd": "kubectl delete -f /chaoslib/pumba/pumba_kube.yml -n shubham", "delta": "0:00:02.351734", "end": "2020-04-13 08:58:03.712020", "rc": 0, "start": "2020-04-13 08:58:01.360286", "stderr": "", "stderr_lines": [], "stdout": "pod \"pumba-sig-kill-0eyqnl\" deleted", "stdout_lines": ["pod \"pumba-sig-kill-0eyqnl\" deleted"]}

2020-04-13 08:58:04.370885 Step: [CleanUP]: Confirm that the pumba pod is deleted successfully 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:126
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -l name=pumba-sig-kill-0eyqnl --no-headers -n shubham", "delta": "0:00:00.443076", "end": "2020-04-13 08:58:04.350844", "rc": 0, "start": "2020-04-13 08:58:03.907768", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}

2020-04-13 08:58:04.983681 Step: [Verify]: Verify the restartCount of application container 
task path: /chaoslib/pumba/pod_failure_by_sigkill.yml:147
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pod nginx-6546cd7545-qhpkc -n shubham -o=jsonpath='{.status.containerStatuses[?(@.name==\"nginx\")].restartCount}'", "delta": "0:00:00.401883", "end": "2020-04-13 08:58:04.963316", "rc": 0, "start": "2020-04-13 08:58:04.561433", "stderr": "", "stderr_lines": [], "stdout": "2", "stdout_lines": ["2"]}
included: /utils/common/status_app_pod.yml for 127.0.0.1

2020-04-13 08:58:05.546866 Step: [Status]: Checking whether application pods are in running state 
task path: /utils/common/status_app_pod.yml:2
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -n shubham -l run=nginx -o custom-columns=:.status.phase --no-headers", "delta": "0:00:00.374216", "end": "2020-04-13 08:58:05.525242", "rc": 0, "start": "2020-04-13 08:58:05.151026", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

2020-04-13 08:58:06.055205 Step: [Status]: Checking whether application containers are in running state 
task path: /utils/common/status_app_pod.yml:9
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pod -n shubham -l run=nginx --no-headers -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | tr ' ' '\\n' | uniq", "delta": "0:00:00.366424", "end": "2020-04-13 08:58:06.034729", "rc": 0, "start": "2020-04-13 08:58:05.668305", "stderr": "", "stderr_lines": [], "stdout": "true", "stdout_lines": ["true"]}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 08:58:06.541014 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.297387", "end": "2020-04-13 08:58:06.520793", "rc": 0, "start": "2020-04-13 08:58:06.223406", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T08:58:06+00:00", "stdout_lines": ["2020-04-13T08:58:06+00:00"]}

2020-04-13 08:58:06.596783 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T08:58:06Z"}, "changed": false}

2020-04-13 08:58:07.113638 Step: [Event]: Creating a PostChaosCheck event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "create", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T08:58:06Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "c3ba3c35-7d64-11ea-8f95-42010a8000cc"}, "kind": "Event", "lastTimestamp": "2020-04-13T08:58:06Z", "message": "AUT is Running successfully", "metadata": {"creationTimestamp": "2020-04-13T08:58:07Z", "name": "event-\"PostChaosCheck\"", "namespace": "shubham", "resourceVersion": "1475", "selfLink": "/api/v1/namespaces/shubham/events/event-%22PostChaosCheck%22", "uid": "e4638f25-7d64-11ea-8f95-42010a8000cc"}, "reason": "PostChaosCheck", "reportingComponent": "", "reportingInstance": "", "source": {"component": "container-kill-72rqv0-fd2qz"}, "type": "Normal"}}

2020-04-13 08:58:07.159838 Step: set_fact 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:96
details => {"ansible_facts": {"flag": "Pass"}, "changed": false}

2020-04-13 08:58:07.211769 Step: [Result]: Getting the final result of container-kill experiment 
task path: /experiments/generic/container_kill/container_kill_ansible_logic.yml:99
details => {
    "msg": "container-kill experiment has been Passed"
}
included: /utils/runtime/getting_failure_step.yml for 127.0.0.1

2020-04-13 08:58:07.479054 Step: [Failure-Detection]: Recording the name of failed step 
task path: /utils/runtime/getting_failure_step.yml:44
details => {"ansible_facts": {"failStep": "N/A"}, "changed": false}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 08:58:07.973952 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.301022", "end": "2020-04-13 08:58:07.945162", "rc": 0, "start": "2020-04-13 08:58:07.644140", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T08:58:07+00:00", "stdout_lines": ["2020-04-13T08:58:07+00:00"]}

2020-04-13 08:58:08.065301 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T08:58:07Z"}, "changed": false}

2020-04-13 08:58:08.921787 Step: [Event]: Creating a Summary event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "patch", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T08:58:07Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "c3ba3c35-7d64-11ea-8f95-42010a8000cc"}, "kind": "Event", "lastTimestamp": "2020-04-13T08:58:07Z", "message": "container-kill Experiment Passed!", "metadata": {"creationTimestamp": "2020-04-13T08:54:05Z", "name": "event-\"Summary\"", "namespace": "shubham", "resourceVersion": "1476", "selfLink": "/api/v1/namespaces/shubham/events/event-%22Summary%22", "uid": "549b7103-7d64-11ea-8f95-42010a8000cc"}, "reason": "Summary", "reportingComponent": "", "reportingInstance": "", "source": {"component": "container-kill-72rqv0-fd2qz"}, "type": "Normal"}}
included: /utils/runtime/update_chaos_result_resource.yml for 127.0.0.1

2020-04-13 08:58:09.885687 Step: [Result]: Update the chaos result CR to reflect EOT (End of Test) 
task path: /utils/runtime/update_chaos_result_resource.yml:23
details => {"changed": true, "checksum": "50625920bbeedb0ce460326b47125bc8a02fc38a", "dest": "/utils/runtime/chaos-result.yml", "gid": 0, "group": "root", "md5sum": "11ea490c58dda1be41ef9c3d7d5cc849", "mode": "0644", "owner": "root", "size": 331, "src": "/root/.ansible/tmp/ansible-tmp-1586768289.2-98159434628313/source", "state": "file", "uid": 0}

2020-04-13 08:58:10.788155 Step: [The END]: Apply the chaos result CR for container-kill experiment 
task path: /utils/runtime/update_chaos_result_resource.yml:33
details => {"changed": true, "cmd": "kubectl apply -f /utils/runtime/chaos-result.yml -n shubham", "delta": "0:00:00.625458", "end": "2020-04-13 08:58:10.767868", "failed_when_result": false, "rc": 0, "start": "2020-04-13 08:58:10.142410", "stderr": "", "stderr_lines": [], "stdout": "chaosresult.litmuschaos.io/engine1-container-kill configured", "stdout_lines": ["chaosresult.litmuschaos.io/engine1-container-kill configured"]}
META: ran handlers
META: ran handlers

2020-04-13 08:58:10.796094 ************ RELAX, EXPERIMENT ENDS! ************ 
127.0.0.1                  : ok=58   changed=27   unreachable=0    failed=0   


```

File: experiments/generic/container_kill/container_kill_ansible_prerequisites.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- name: Identify the chaos util to be invoked 
+- name: "[PreReq] Identify the chaos util for {{ c_experiment }} experiment"
```
Comment (uditgaurav) [inline experiments/generic/container_kill/container_kill_ansible_prerequisites.yml:1]: How about using Action or Chaos or something like that in place of PreReq?

File: experiments/generic/pod_delete/pod_delete_ansible_logic.yml [hunk 2]
```diff
@@ -50,7 +50,7 @@
               - "Label        : {{ a_label }}"
               - "Ramp Time    : {{ ramp_time }}"  
      
-        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+        ## PRE-CHAOS APPLICATION RUNNING STATUS CHECK
```
Comment (uditgaurav) [inline experiments/generic/pod_delete/pod_delete_ansible_logic.yml:53]: `## PRE-CHAOS APPLICATION STATUS CHECK`

File: experiments/generic/pod_delete/pod_delete_ansible_logic.yml [hunk 3]
```diff
@@ -75,7 +75,7 @@
             app_ns: "{{ a_ns }}"
             app_label: "{{ a_label }}"
    
-        ## POST-CHAOS APPLICATION LIVENESS CHECK
+        ## POST-CHAOS APPLICATION RUNNING STATUS CHECK
```
Comment (uditgaurav) [inline experiments/generic/pod_delete/pod_delete_ansible_logic.yml:78]: `## PRE-CHAOS APPLICATION STATUS CHECK`

**Pull request merged**